analyze/Cargo.toml

```diff
@@ -15,3 +15,4 @@ path = "./analyze.rs"
 twiggy-ir = { version = "0.1.0", path = "../ir" }
 twiggy-opt = { version = "0.1.0", path = "../opt", default-features = false }
 twiggy-traits = { version = "0.1.0", path = "../traits" }
+petgraph = "0.4.12"
```

Comment:
Right now we depend on an older point release in `ir/Cargo.toml`:

```
ir/Cargo.toml:17:petgraph = "0.4.11"
```

Can you update that so that we have the same dependency? Thanks!